testing/N_00014_0002/test_N_00014_0002.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pytest
+import numpy as np
+from uegccdutils import Dataset
+
+dir = os.path.basename(__file__)[5:-3] # infer directory from test name
+
+def test_output(data_regression):
+	data = Dataset(dir).test_summary()
```

Comment:
We use spaces rather then tabs, 4 spaces = 1 tab. If you use VIM let me know and I can send the vimrc setting to change this.

Reply:
I'll switch it in VSCode! It might be good to write a "contributing" document to note such style preferences